src/components/localization/localized-link.tsx

```diff
@@ -82,24 +82,13 @@ export const SharedLinkStyle = css<SharedLinkStyleProps>`
         `}
 `
 
-const ShareDisabledStyle = css<{ disabled: boolean }>`
-    ${({ disabled }) =>
-        disabled &&
-        `
-        pointer-events: none;
-        opacity: 0.32;`}
-`
```

Comment:
this is the way that we can disable an `a` tag.
what we need is to just remove the unallowed `disabled` attribute on the element. we still need to use this pointer-events  and opacity here.

Reply:
Ako, it's not `attribute`, its custom `props` and it's unused code. We don't use it anywhere else.

Reply:
it was named `disconnected` and in this case it confused everyone